src/lib.rs

```diff
@@ -251,4 +267,146 @@ mod test {
         let data: u16 = 0b1000_0000_0000_0001;
         assert!((single_ended_rdata_to_scaled_voltage(data as i16) - -V_MAX).abs() < EPS);
     }
+
+    const DEFAULT_CONFIG: u8 = 0b0000_0000;
+    // Since the only bit that is checked is the MSB
+    // the default status should have MSB == 0
+    const NOT_READY_STATUS: u8 = !0b1000_0000;
+    const DEVICE_ADDRESS: u8 = 0b0000_0000;
+
+    fn new_ads1119(transactions: &[I2cTransaction]) -> Ads1119<I2cMock> {
+        let device_address = 0;
+        Ads1119::new(I2cMock::new(transactions), device_address)
+    }
+
+    fn destroy_ads1119(device: Ads1119<I2cMock>) {
+        device.destroy().done();
+    }
+
+    // run "done" on the device but ignore if all
+    // expectations were not consumed
+    fn destroy_ads1119_silently(device: Ads1119<I2cMock>) {
```

Comment:
This method is a bit hacky but I can't see a better way to ignore expectations for cases where we're not testing the exact expected communication between this driver and the device but also can't know with 100% certainty what the expected transactions will be (because of timing unpredictability).

Reply:
I think this works for our intended purpose.